platform-hub-api/spec/integration/auditing_integration_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
```

Comment:
I wanted these particular spec files to stand out a little, because it goes outside of the boundaries of the "standard" spec groups that rspec-rails provides (the `integration` folder is non-standard). In this particular case I wanted a place for any kind of integration spec and so figured naming the actual files with `*_integration_spec.rb` would help later.